libvirt/resource_libvirt_domain_test.go

```diff
@@ -1767,3 +1767,58 @@ func testAccCheckLibvirtDomainDestroy(s *terraform.State) error {
 	}
 	return nil
 }
+func TestAccLibvirtDomain_FwCfgName(t *testing.T) {
+	var domain libvirt.Domain
+	var volume libvirt.StorageVol
+	randomDomainName := acctest.RandStringFromCharSet(10, acctest.CharSetAlpha)
+	randomFwCfgName := acctest.RandStringFromCharSet(10, acctest.CharSetAlpha)
+	randomPoolName := acctest.RandStringFromCharSet(10, acctest.CharSetAlpha)
+	randomPoolPath := "/tmp/terraform-provider-libvirt-pool-" + randomPoolName
+	randomIgnitionName := acctest.RandStringFromCharSet(10, acctest.CharSetAlpha)
+	var config = fmt.Sprintf(`
+	data "ignition_systemd_unit" "acceptance-test-systemd" {
+		name    = "example.service"
+		content = "[Service]\nType=oneshot\nExecStart=/usr/bin/echo Hello World\n\n[Install]\nWantedBy=multi-user.target"
+	}
+
+	data "ignition_config" "acceptance-test-config" {
+		systemd = [
+		"${data.ignition_systemd_unit.acceptance-test-systemd.id}",
+		]
+	}
+
+    resource "libvirt_pool" "%s" {
```

Comment:
for this test, I don't think we need the `libvirt_pool` Just remove this resource and adapt it accordingly.

Having the libvirt_pool here creation would add a pointer of failure of the test.

Reply:
If we remove the `livbirt_pool` libvirt will looking for the `default` pool which leads to CI failure. :thinking:

Reply:
ok somehow I think is related to the fact we don't have a default. I think in that case we can restore the default pool

Reply:
Ok, I revert the commit so.